src/CloudVolume.jl

```diff
@@ -30,45 +30,65 @@ end
 CachedVolume = cached(cv.CloudVolume)
 CachedStorage = cached(cv.Storage)
 
-immutable CloudVolumeWrapper
+struct CloudVolumeWrapper 
 	val
+    is1based:: Bool
 	function CloudVolumeWrapper(storage_string; mip=0,
 						bounded=true,
 						fill_missing=false,
                         cache=false,
                         progress=false,
-						info=nothing)
+						info=nothing,
+                       is1based=false)
 		return new(CachedVolume(storage_string, mip, 
 						bounded, 
 						fill_missing,
                         cache,
                         progress,
-						info))
+						info), is1based)
 	end
 end
 
 function Base.getindex(x::CloudVolumeWrapper, slicex::UnitRange, 
                                         slicey::UnitRange, slicez::UnitRange)
-    return squeeze(get(x.val, 
-            (pyslice(slicex.start,slicex.stop+1),
-            pyslice(slicey.start,slicey.stop+1),
-            pyslice(slicez.start,slicez.stop+1))),4)
+    t = time()
+    if x.is1based
```

Comment:
@macrintr this will still allow negative indexing, just different indexing style.